js/renovation/pager/pager-props.tsx

```diff
@@ -0,0 +1,38 @@
+import { ComponentBindings, OneWay, TwoWay } from 'devextreme-generator/component_declaration/common';
+import messageLocalization from '../../localization/message';
+
+@ComponentBindings()
+export default class PagerProps {
+  // Vitik: Renovation workaround for preact remove all existed element attribute from root node
+  @OneWay() elementAttr?: any;
+
+  @OneWay() showInfo?: boolean = false;
+
+  @OneWay() infoText?: string = messageLocalization.getFormatter('dxPager-infoText')();
```

Comment:
Only default english localization works because PagerProps is generated for React to static field defaultProps of Pager component. To fix the issue generated code should be changed or getFormatter should be called during render.

Reply:
Thank you I add it in my TODO list